targets/targets.json

```diff
@@ -14712,13 +14712,12 @@
             "5"
         ]
     },
-    "NUMAKER_IOT_M263A": {
+    "MCU_M261": {
```

Comment:
As noted in my comment, if this is base MCU, needs to be private

Reply:
@0xc0170 Fixed it. Thanks for your explanation.